config/routes.rb

```diff
@@ -1,13 +1,9 @@
 Rails.application.routes.draw do
```

Comment:
### Optional
- [ ] I have run the app successfully but nothing is getting rendered on the browser. But am making it optional because I don't know if it's on my end or a bug in the code. Other code reviewers can give a second opinion about it

Reply:
## It works for me fine

![image](https://user-images.githubusercontent.com/58692590/151454820-ad555a20-4f88-4012-8ffe-5b653da97c4c.png)

![image](https://user-images.githubusercontent.com/58692590/151455006-1256ad7b-e6cb-4f58-98fd-c3a9963719f4.png)

![image](https://user-images.githubusercontent.com/58692590/151455102-5d246e5a-5206-462b-b4ff-cafcff937b59.png)

![image](https://user-images.githubusercontent.com/58692590/151455152-ce889e36-03d6-45a6-aca8-90fdb7dcfcf9.png)